checkov/main.py

```diff
@@ -401,6 +402,8 @@ def add_parser_args(parser):
     parser.add('--skip-cve-package',
                help='filter scan to run on all packages but a specific package identifier (denylist), You can '
                     'specify this argument multiple times to skip multiple packages', action='append', default=None)
+    parser.add('--prisma_api_url',
```

Comment:
```suggestion
    parser.add('--prisma-api-url',
```
to keep it in line with the other parameters 🙂

Reply:
Done.